config/routes.rb

```diff
@@ -45,6 +45,8 @@
       as: :openid_connect_configuration
 
   get '/account' => 'accounts#show'
+  get '/account/reactivate/start' => 'reactivate_account#index', as: :manage_reactivate_account
+  put 'account/reactivate/start' => 'reactivate_account#update'
```

Comment:
1. should this get a leading slash for consistency?
2. to keep these alphabetized by path, should we move these new ones down 2 lines?